recipes/alphacsc/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set version = "0.3" %}
+
+package:
+  name: alphacsc
+  version: {{ version }}
+
+source:
+  url: https://github.com/alphacsc/alphacsc/archive/{{ version }}.tar.gz
+  sha256: 008acfcd39e8f9b926a22535015ae09a4e6e85986d496d4b279178b396a1dda2
+  patches:
+   - patches/001-remove_sporco_files_from_package.patch
+
+build:
+  number: 0
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+    - cython                                 # [build_platform != target_platform]
+    - numpy                                  # [build_platform != target_platform]
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - cython
+  run:
+    - python
+    - mne
+    - numba
+    - numpy
+    - scipy
+    - joblib
+    - matplotlib-base
+    - scikit-learn
+
+test:
+  imports:
+    - alphacsc
+  requires:
+    - pip
+    - pytest
```

Comment:
You don't really want to run an entire test suite here. Just testing whether some basic imports work is sufficient.

Reply:
Oh and the `pip check` of course is always a good idea.